packages/@magic-ext/react-native-bare-oauth/README.md

```diff
@@ -1,5 +1,67 @@
- 
 
- 
 
 
+
+
+# 🔒 Magic OAuth Extension for React Native (Bare)
+
+[![<MagicLabs>](https://circleci.com/gh/magiclabs/magic-js.svg?style=shield)](https://circleci.com/gh/magiclabs/magic-js)
+
+> With the Magic JavaScript SDK OAuth extension, you can plug into your favorite social login providers with one, easy-to-use API.
+
+<p align="center">
+  <a href="https://github.com/magiclabs/magic-js/blob/master/packages/@magic-ext/oauth/LICENSE">License</a> ·
+  <a href="https://github.com/magiclabs/magic-js/blob/master/packages/@magic-ext/oauth/CHANGELOG.md">Changelog</a> ·
+  <a href="https://github.com/magiclabs/magic-js/blob/master/CONTRIBUTING.md">Contributing Guide</a>
+</p>
+
+## 📖 Documentation
+
+See the [developer documentation](https://magic.link/docs/social-login) to learn how to get started with OAuth in Magic SDK.
+
+## 🔗 Installation
+
+Integrating your app with OAuth will require our client-side NPM package and OAuth extension:
+
+```bash
+# Via NPM:
+npm install --save @magic-ext/react-native-bare-oauth
```

Comment:
nit: we should probably include the `@magic-sdk/react-native-bare` + `expo` here to make it clear that this requires the package to work properly. In other words, dummy-proof it ;)

Reply:
Thing is we also need the peer deps as well for it to work well, that's why I decided to add the warning below instead. That way we have one source of truth instead of having them on both READMEs and risking having inconsistencies.